lib/components/card-list/card-list-item.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { ComponentPropsWithoutRef } from "react";
+import cn from "classnames";
+
+import "./styles.scss";
+
+export const CardListItem = ({
```

Comment:
I would rename this to just Card, so its not so coupled to the CardList component

Reply:
Also please look at the isomorphic approach for this component.